doc_user/UserManual.adoc

```diff
@@ -801,6 +801,288 @@ image:image251.png[success]
 
 NOTE: This button can be enabled or disabled to control movements deletions. Ask the Administrator or check the Administrator's Guide for more information.
 
+=== 4.4 Inventory ([.underline]##I##nventory)
+
+==== 4.4.1 Main Characteristics
+
+The Inventory features is used to correct the stock in the *_Main Store_*.
+
+==== 4.4.2 Functions of Inventory
+
+To access the functions of inventory, click on *[.underline]##I##nventory* on the Pharmacy menu.
+
+image:image22.png[Phamacy Menu,width=220,height=182]
+
+All functions available under *[.underline]##I##nventory* are accessible from the *_Inventory Browser_* window. By default, the system shows all the inventories recorded in the period.
+
+The *_Inventory Browser_* window is composed of three areas: the *Search panel* at the top, the *Inventory table* in the center and the *Buttons panel* on the bottom. See the image below.
+
+The *Search panel* at the top, allows narrowing or sub-setting the inventory list by:
+
+* *Date From* and *Date To*: dates selectors used to select a period.
+* *[status]* selector (e.g., 'all'): inventory status filter.
+
+The *Inventory table* in the center identifies the inventory by:
+
+* *Reference*: indicates the reference of the inventory.
+* *Date*: indicates the date with which the inventory was entered into the system.
+* *Status*: indicates the status of the inventory.
+* *User*: indicates the user who entered the inventory.
+
+The *Buttons panel* has all the functions of the main inventory, the following choices are available:
+
+* *[.underline]##N##ew*: create a new inventory.
+* *[.underline]##E##dit*: modify an existing inventory.
+* *[.underline]##V##iew*: show an existing inventory.
+* *[.underline]##D##elete*: cancel an existing inventory.
+* *[.underline]##C##lose*: close the window and return to the main menu.
+
+image:image252.png[Inventory Browser]
+
+[#create-a-new-inventory]
+===== 4.4.2.1 Create a New Inventory ([.underline]##N##ew Inventory)
+
+The *New Inventory* function is used to create a new inventory on the main store. Click on *[.underline]##N##ew* button in the *_Inventory Browser_* window to access the *_New Inventory_* window.
+
+The *_New Inventory_* window is composed of three areas: the *Inventory panel* at the top, the *Inventory row table* in the center and the *Buttons panel* on the bottom. See the image below.
+
+image:image253.png[New Inventory]
+
+The data shown in the *Inventory panel* area are:
+
+* *Date*: the date where the inventory was entered at the system.
```

Comment:
```suggestion
* *Date*: the date when the physical check for this inventory has been done in the main store.
+
NOTE: this will affect the _validation_ step, when the system will check all further charges and/or discharges recorded after this date and so potentially affecting the theoretical and real quantities in the inventory
```